tests/numpy/test_arithmetic.py

```diff
@@ -147,6 +147,41 @@ def test_inner_outer(nps_app_inst):
 
 # TODO(hme): Add broadcast tests.
 
+def test_broadcast_block_shape_error(nps_app_inst):
+    from nums import numpy as nps
+    import pytest
+    assert nps_app_inst is not None
+
+    _ops = ['add', 'subtract', 'divide', 'bitwise_and']
+
+    def check_broadcast_block_shape_mismatch_simple_error(_ns_a, _ns_b,
```

Comment:
Could you shorten this name?

Reply:
Agree. Since this is a nested function, we could rename it to `check_value_error`.